### DISCUSSION TITLE
Default Browser per domain or URL

### DISCUSSION THREAD

**Discussion by N-Molham** [will-stone/browserosaurus#641] [Ideas]
Love the app and it is pretty helpful.

I use [BrowseRouter](https://github.com/slater1/BrowseRouter) for my Windows laptop which has a pretty nice feature that allows links with matching pattern to auto-open in the corresponding browser.

Ref: https://github.com/slater1/BrowseRouter#config

It would be amazing if something like that can be implemented here.

**Comment by will-stone** [comment]
Hi. Thanks.

Duplicate: https://github.com/will-stone/browserosaurus/discussions/476
